microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,187 @@
+| IE Mode | Off | No | This feature is turned off.  |  
+| Immersive Reader | Off | No | This feature is dependent on the browser UI for control and triggering.  This feature is turned off.  |  
+| Intrusive Ads | Off | No | This feature is turned off.  |  
+| Read Aloud | Off | No | This feature is turned off.  |  
+| Smart Screen | On* | No | The UI for this feature is removed, however the underlying functionality persists.  Additionally, you may turn off Smart Screen using command-line switch.  |  
+| Translate | Off | No | This feature is turned off.  |  
+| Tracking Prevention | On* | No | *The UI for this feature is removed, however the underlying functionality persists.  Tracking prevention is always set to balanced.|  
+| Profile and Identity | Off | No | The feature is turned off that syncs your favorites, cookies, and so on, between machines.  |  
+| Web Payment API | Off | No | This feature is turned off.  | 
+| Windows Defender Application Guard | Off | No | This feature is turned off.  |  
+| edge:// URLs | Review Details | No | Settings for the browser are displayed on webpages that use internal URLs.  Most of the webpages have Microsoft Edge branding or don't make sense within the context of WebView2.  For more details about the internal URLs that are turned off, navigate to [Blocked internal URLs](#blocked-internal-urls).  |  
+
+### Blocked internal URLs  
+
+The following built-in Microsoft Edge and Google Chrome webpages aren't available in WebView2.  
+
+*   `chrome-search://local-ntp/local-ntp.html`  
+*   `edge://application-guard-internals`  
+*   `edge://apps`  
+*   `edge://compat`  
+*   `edge://extensions`  
+*   `edge://favorites`  
+*   `edge://help`  
+*   `edge://management`  
+*   `edge://network-error`  
+*   `edge://new-tab-page`  
+*   `edge://newtab`  
+*   `edge://omnibox`  
+*   `edge://settings`  
+*   `edge://supervised-user-internals`  
+*   `edge://version`  
+
+### Additional shortcuts information  
+
+Key bindings are subject to change when the underlying browser updates.  An accelerator key that is turned off by default may turn on if the feature is supported in WebView2.  However, it's guaranteed that if you set `AreBrowserAcceleratorKeysEnabled` to `FALSE`, all keys that access browser-like features are turned off, and all basic text-editing and movement shortcuts remain turned on.  
+
+The following shortcuts are always turned off in WebView2.
+
+An asterisk \(`*`\) character indicates the shortcut is technically not turned off, but the feature it accesses is turned off or doesn't apply to WebView.  
+
+| Action | Windows |  
```

Comment:
In the following tables you name the second column 'Windows'. Should it be 'Windows shortcut key'?

Reply:
The entire section defines shortcuts and matches the content style located on https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/shortcuts/index.